thrall/test/resources/logback-test.xml

```diff
@@ -8,11 +8,11 @@
         </encoder>
     </appender>
 
-    <logger name="play" level="INFO" />
-    <logger name="application" level="DEBUG" />
+<!--  <logger name="application" level="INFO" />-->
+<!--  <logger name="play" level="INFO" />-->
 
-    <logger name="com.amazonaws">
-        <level value="ERROR" />
-    </logger>
+<!--  <root level="INFO">-->
```

Comment:
Should we leave this commented out?

Reply:
I think it's sometimes useful locally when debugging, otherwise it's too noisy to uncomment. Happy to uncomment if you think the test logs are helpful.

Reply:
Nope that's fine just checking for typos 👍